samples/cellular/nrf_cloud_multi_service/src/application.c

```diff
@@ -380,3 +383,14 @@ void main_application_thread_fn(void)
 		k_timer_status_sync(&sensor_sample_timer);
 	}
 }
+
+void test_counter_enable_set(const bool enable)
+{
+	LOG_DBG("Test counter %s", enable ? "enabled" : "disabled");
+	test_counter_enabled = enable;
+}
+
+bool test_counter_enable_get(void)
+{
+	return test_counter_enabled;
```

Comment:
Can this be `test_counter_enabled || IS_ENABLED(CONFIG_TEST_COUNTER)` with a comment explaining that we do this so that we have a reliable way to force the test counter to be enabled during testing?

Alternatively, we could create a new KConfig `CONFIG_TEST_COUNTER_ALWAYS` which has the force-enable behavior

Reply:
your concern is that the test counter is disabled in the reported section of the shadow and then we update the device with new firmware which enables the test counter, but then that will be overridden by the shadow?

we have the ability to reliably update the shadow to mitigate this concern.

Reply:
I guess I just want to be able to run builds with the test counter enabled without having to even worry about the shadow

Reply:
if you feel that `CONFIG_TEST_COUNTER_ALWAYS` provides value to the customer, I will add it.

Reply:
It would provide value to me -- the same value which CONFIG_TEST_COUNTER was originally added to provide

Reply:
Not to muddy the water here, but, an alternative to controlling the test counter this way might be to control the LED pattern or a specific LED.